target/linux/ipq40xx/image/Makefile

```diff
@@ -221,6 +221,30 @@ define Device/qcom_ap-dk04.1-c1
 endef
 TARGET_DEVICES += qcom_ap-dk04.1-c1
 
+define Device/qxwlan_e2600ac-c1
+	$(call Device/FitImage)
+	DEVICE_TITLE :=  Qxwlan E2600AC-C1 (nor flash)
+	BOARD_NAME := e2600ac-c1
+	DEVICE_DTS := qcom-ipq4019-qxwlan-e2600ac-c1
+	KERNEL_SIZE := 4096k
+	IMAGE_SIZE := 26624k
```

Comment:
This looks like it was copied from the glinet_gl_b1300, wasn't it? 

It's a bit odd that both the qxwlan_e2600ac-c1 and the glinet_gl_b1300 specify the same IMAGE_SIZE of 26624k = "0x1a00000" whereas the firmware partition is specified for 0x1e80000. If you want you could make the IMAGE_SIZE a bit bigger still. That said, if you want to leave it as is, that's fine too. Although, if there was a specific reason for the decision, can you please tell me what/why?